keyboards/doio/kb04/info.json

```
@@ -0,0 +1,111 @@
+    "rgb_matrix": {
+        "animations": {
+            "solid_color": true,
+            "alphas_mods": true,
+            "gradient_up_down": true,
+            "gradient_left_right": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "rainbow_moving_chevron": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_fractal": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "typing_heatmap": true,
+            "digital_rain": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [0, 0], "x": 0, "y": 0, "flags": 1},
+            {"matrix": [0, 1], "x": 75, "y": 0, "flags": 1},
+            {"matrix": [0, 2], "x": 149, "y": 0, "flags": 1},
+            {"matrix": [0, 3], "x": 224, "y": 0, "flags": 1},
+            {"matrix": [0, 4], "x": 112, "y": 64, "flags": 1},
+            {"matrix": [0, 5], "x": 112, "y": 64, "flags": 1},
+            {"matrix": [0, 6], "x": 112, "y": 64, "flags": 1},
+            {"matrix": [0, 7], "x": 112, "y": 64, "flags": 1}
+        ],
+        "max_brightness": 200,
+        "sleep": true
+    },
+    "url": "https://www.keebmonkey.com/products/megalodon-macro-pad-with-a-knob",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0401",
+        "vid": "0xD010"
+    },
+    "ws2812": {
+        "pin": "A10"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label": "1!", "matrix": [0, 0], "x": 0, "y": 0},
+                {"label": "2@", "matrix": [0, 1], "x": 1, "y": 0},
+                {"label": "3#", "matrix": [0, 2], "x": 2, "y": 0},
+                {"label": "4$", "matrix": [0, 3], "x": 3, "y": 0},
+                {"label": "Encoder", "matrix": [0, 4], "x": 4.75, "y": 0}
```

Does not match physical layout
```suggestion
                {"label": "Encoder", "matrix": [0, 4], "x": 1.5, "y": 1}
```